apps/condo/domains/ticket/access/TicketComment.js

```diff
@@ -46,11 +46,12 @@ async function canManageTicketComments ({ authentication: { item: user }, origin
 }
 
 async function canSetUserField ({ authentication: { item: user }, originalInput, addFieldValidationError }) {
+    if (!user) return false    
     if (user.isAdmin) return true
-    if (get(originalInput, ['user', 'connect', 'id']) !== user.id) {
-        return false
+    if (get(originalInput, ['user', 'connect', 'id']) === user.id) {
```

Comment:
I think we can simply 
`return get(originalInput, ['user', 'connect', 'id']) === user.id)`

Reply:
it seems that we have not checked all cases here and we will have to add some code here in the future. And it's easier to leave like so